module1/homework/removeVowels/vowel.cpp

```diff
@@ -0,0 +1,14 @@
+#include "vowel.hpp"
+
+#include <algorithm>
+#include <cctype>
+
+void removeVowels(std::vector<std::string>& words) {
+    std::vector<char> vowels{'a', 'e', 'i', 'o', 'u', 'y'};
+    std::for_each(words.begin(), words.end(), [&vowels](std::string& word) {
+        word.erase(std::remove_if(word.begin(), word.end(), [&vowels](char letter) {
+            letter = tolower(static_cast<unsigned char>(letter));
+            return std::find(vowels.begin(), vowels.end(), letter) != vowels.end();
+        }), word.end());
+    });
+}
```

Comment:
new line.
In visual studio code you can set that new line is added during file saving. See
https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files